src/AppPrintOnly.test.tsx

```diff
@@ -300,6 +302,39 @@ it('VxPrintOnly flow', async () => {
   // Check Printed Ballots Count
   getAllByTextWithMarkup('Ballots Printed: 3')
 
+  // Blank Ballot should print
```

Comment:
Lets be more specific: 
```suggestion
  // Card with no votes prints ballot with no votes
```

Reply:
I strongly prefer "blank ballot" because that is the technical term for a ballot without any options selected.

Reply:
```
  // Card with no votes prints a blank ballot with no votes
```
?

Reply:
That sounds incorrect to me --> a card with no votes sounds like a card that wasn't used. A card that contains a blank ballot should print, I maintain that the original comment is correct :)